lib/matplotlib/afm.py

```diff
@@ -91,7 +91,7 @@ def _sanity_check(fh):
     # do something else with the file.
     pos = fh.tell()
     try:
-        line = fh.readline()
+        line = next(fh)
```

Comment:
I think the gain in legibility with .readline() doesn't compensate the brittleness of only being able to pass in a file with an empty readahead buffer.